incubator/hnc/internal/reconcilers/hnc_config.go

```diff
@@ -24,6 +25,10 @@ import (
 	"sigs.k8s.io/multi-tenancy/incubator/hnc/internal/forest"
 )
 
+// HNCConfigReconcilerSingleton stores a pointer to the cluster-wide config reconciler so anyone can
```

Comment:
For "anyone", do we intent code outside `reconcilers` package to call this?

Reply:
(As discussed in another comment) nope :)